spec/features/restrooms_spec.rb

```diff
@@ -100,7 +101,7 @@
 
       click_on "Preview"
 
-      expect(page).to have_css("div#mapArea", visible: :visible)
+      page.has_css?(".nearby-container .listItem", visible: :visible)
```

Comment:
I think the original CSS selector was correct here. (New one was probably copy-pasted from a nearby spec by accident?)

Looking to revert this soon if that's okay.

(Already said it elsewhere, but thanks again for fixing all this, and the dependency updates are appreciated as well.)